mne/tests/test_epochs.py

```diff
@@ -57,6 +59,71 @@
 rng = np.random.RandomState(42)
 
 
+def test_handle_duplicate_events():
+    """Test handling of duplicate events."""
+    # A general test case
+    EVENT_ID = {'aud': 1, 'vis': 2, 'foo': 3}
+    EVENTS = np.array([[0, 0, 1], [0, 0, 2],
+                       [3, 0, 2], [3, 0, 1],
+                       [5, 0, 2], [5, 0, 1], [5, 0, 3],
+                       [7, 0, 1]])
```

Comment:
```suggestion
                       [5, 0, 3], [7, 0, 1]])
```

Reply:
This was added by Alex, and I think it's for clarity that each row is at a new time sample